### PR TITLE
docs: add NuGet badge to readme for easy discovery

### PR DESCRIPTION
**Pull request by jobullen**
Add the NuGet badge for the Feature Management package along with badge. This resolves issue #151

### CODE REVIEW

**Comment by jobullen** [comment]
> Can you add a badge for the aspnet core package as well?

Added badge.

File: README.md [hunk 1]
```diff
@@ -20,10 +20,19 @@ Here are some of the benefits of using this library:
 
 **API Reference**: https://go.microsoft.com/fwlink/?linkid=2091700
 
+### Packages
+
+| Namespace                              | Package                                                                                                                              |
```
Comment (WeihanLi) [inline README.md:25]: Maybe `Project` is better?

Comment (jimmyca15) [reply README.md:25]: Probably doesn't need to be a table actually. Just a list, the package name and whatever we put for namespace/project will be the same.

Comment (jimmyca15) [reply README.md:25]: @jobullen do you mind making these changes?

**Pull request closed without merging**